docs/contributing/dependencies/index.md

```diff
@@ -55,12 +92,165 @@ version is **13 days**, and **13%** of repositories have adopted this version. R
+    to group these together to avoid errors when building. Sometimes we do this for all update
+    types, and sometimes we only do this for major updates, depending on when we feel like
+    inter-version dependencies exist.
+- Group by **domain** or by **package manager** to reduce the noise of too many PRs
+  - This is a [trade-off](https://docs.renovatebot.com/noise-reduction/#noise-reduction), and we
+    acknowledge that doing so will make identification of any issues stemming from the update
+    harder.
+  - Sometimes we only do this for minor or patch updates, as we acknowledge the trade-off with
+    identifying breaking changes and want to have separate PRs for major updates.
+
+:::info Grouping and team ownership
+
+Grouping does not equal team ownership. In other words, a PR that has dependencies owned by multiple
+teams would have multiple reviewers on it. As a matter of practice, we try to assign all grouped
+dependencies to a single team, so that only a single team is responsible for the review.
+
+:::
+
+### Dependency update type strategies
+
+:::info What are update types?
+
+“Update type” refers to the classification of a version as major, minor, or patch.
+
+:::
+
+### Combining update types
+
+For all of our dependencies, we default to:
+
+1. Combining minor and patch updates into a single PR for a given dependency, and
+2. _Not_ combining major updates with any minor or patch updates for a given dependency
+
+We do this because we want to reduce the noise of separate minor and patch updates, but we also
+don’t want to miss out on incremental minor/patch updates while we do the larger body of work for a
+major update. For example, if we’re updating Angular to version 20, we would want to receive minor
+and patch updates for version 19 while the PR for the version 20 update is open and being evaluated.
+
+This combination of updates is for an individual dependency. This is distinct from grouping multiple
+dependencies into a single PR, which is covered in
+[Dependency grouping strategies](#dependency-grouping-strategies).
+
+### Managing patch updates
+
+In order to maintain a sustainable number of update PRs for our teams to review, we have elected to
+send most patch updates through Renovate's
+[Dependency Dashboard Approval](https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval)
+flow. This means that Renovate will _not_ automatically generate a PR for these updates. Rather, it
+will list them in a "Pending Approval" section of the Dependency Dashboard. If a team desires to
+update the dependency to pull in a bug fix that affects our code, they can trigger the pull request
+to be created from the dashboard.
+
+### Dependency reviewer assignment strategies
+
+In order for a dependency update PR to successfully move through the review process, it must be
+assigned a reviewer. We do this one of two ways:
+
+1. Through GitHub `CODEOWNERS`
```

Comment:
💭 This feels like an unordered list to me.

Reply:
Addressed with cf9320b75d9d56cf5291b3cf3b554bce77dda3e5.